src/python/utils/get_stop_words.py

```diff
@@ -0,0 +1,73 @@
+from index_storage import IndexStorage
+from index_partition import IndexPartition
+
+from stopword_controller import StopWordController
+from stopword_model import StopWordModel
+
+import boto3
+
+
+'''
+Return all partion keys from dynamodb
+'''
+def get_s3_keys():
```

Comment:
Could you move this functionality to the IndexStorage class?

Reply:
I believe I am doing a scan?

Reply:
Oh, you're correct. For some reason I thought you were doing s3.list_keys. In this case could you move it to the IndexController class?